dracut/30ignition/ignition-setup-base.service

```diff
@@ -5,9 +5,8 @@ ConditionPathExists=/etc/initrd-release
 DefaultDependencies=false
 Before=ignition-complete.target
 
-Before=local-fs-pre.target
```

Comment:
AFAICT, this dates back from the introduction of the setup service in CL. I don't think we actually need a hard ordering against `local-fs-pre.target` nowadays though.